README.adoc

```diff
@@ -20,3 +19,63 @@ Assuming that you already have a Trino server set up.
 === Tests
 
 The tests assume that OPA is available on your path as `opa`, and can then be executed by running `mvn test`.
+
+=== Example
+
+The example `example.rego` allows the `admin` user to do everything. User `bob` is limited to run queries and access catalogs.
+
+----
```

Comment:
Having an include of `example.rego` would be great

Reply:
done